app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -99,6 +99,8 @@ public static MediaDetailFragment forMedia(int index, boolean editable, boolean
     MediaDetailSpacer spacer;
     @BindView(R.id.mediaDetailTitle)
     TextView title;
+    @BindView(R.id.mediaDetailCaption)
```

Comment:
Please follow the underscore based convention media_detail_caption, I know the rest of the values are not following that, but let us do it.

Reply:
Done